src/vivarium_nih_us_cvd/components/risks.py

```diff
@@ -148,11 +163,11 @@ def _get_current_exposure(self, index: pd.Index) -> pd.Series:
         continuous_exposure = self.continuous_exposure(index)
 
         bins = [
-            RISK_EXPOSURE_LIMITS["high_systolic_blood_pressure"]["minimum"],
+            0,
```

Comment:
Would we ever have <0 at this point?

Reply:
Nope, unless something has gone very wrong. But then attempting to call the riskeffect target modifier will throw an error.